.github/workflows/ruby_tests.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         run: bundle exec rake spec
   test_ruby:
     runs-on: ubuntu-latest
-    needs: rubocop
+    # needs: rubocop
```

Comment:
Yeah, I've got no idea what's wrong with rubocop :(

Reply:
I have no idea either, I'm investigating

Reply:
I'll take a look into that in follow-up